### ISSUE TITLE
[Feature request] Generate badge graphics

### ISSUE THREAD

**Issue by ulrichb** [danielpalme/ReportGenerator#25]
It wold be cool if _ReportGenerator_ additionally generates badge graphics for the total statement / branch coverage, like the following example.
![sample](https://cloud.githubusercontent.com/assets/388796/9551924/994290e6-4db4-11e5-8fc7-394307ae77ea.PNG)

Use case: These graphic files could be uploaded somewhere (or directly hosted by the build server) so that they can be referenced e.g. in a GitHub README page.

Ideally these graphics are generated as SVG to have no problems with background colors / scaling.

**Comment by danielpalme** [comment]
Great idea. Will add this soon.

**Comment by arpitgold** [comment]
+1 :+1: 
It will be great add to the functionality

**Comment by danielpalme** [comment]
Please have a look at the new released version 2.3.0.0.
By adding *Badges* to the *-ReportTypes* you can create some SVG badges

Sample command line arguments:
```
ReportGenerator.exe "-reports:coverage.xml" "-targetdir:coveragereport" -reporttypes:Html;Badges
```

Sample badge:
![Sample badge](http://danielpalme.github.io/ReportGenerator/resources/badge.svg)

**Comment by ulrichb** [comment]
Just tested it. Great job! Looks really nice.
